tests/wsl/install_from_MSStore.pm

```diff
@@ -26,27 +29,46 @@ sub run {
         cmd => "Enable-WindowsOptionalFeature -Online -FeatureName VirtualMachinePlatform -NoRestart",
         timeout => 300
     );
+    $self->run_in_powershell(
+        cmd => 'Disable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V-Hypervisor -NoRestart'
+    ) if (get_var('WSL2'));
 
     # Reboot and wait for it
     $self->reboot_or_shutdown(1);
     $self->wait_boot_windows;
     $self->open_powershell_as_admin;
 
-    # $self->install_wsl2_kernel;
+    if (get_var('WSL2')) {
+        $self->install_wsl2_kernel;
+    }
+    else {
+        $self->run_in_powershell(
+            cmd => "wsl --set-default-version 1"
+        );
+    }
+
```

Comment:
It would be better to change the settings only when needed.

Reply:
You're right. Changed.